build.sbt

```diff
@@ -178,10 +178,6 @@ def crossSparkSettings(): Seq[Setting[_]] = getSparkVersion() match {
     Antlr4 / antlr4Version := "4.9.3",
     Test / javaOptions ++= Seq("-Dlog4j.configurationFile=log4j2.properties"),
 
-    // Java-/Scala-/Uni-Doc Settings
```

Comment:
This has to be moved to `Unidoc` settings because when unidoc is disabled this fails compilation since it's an unrecognized option.